### DISCUSSION TITLE
Localhost shows site's folder structure and filenames. Also shows php code on web page's content

### DISCUSSION THREAD

**Discussion by chinthyelim** [shivammathur/homebrew-php#862] [Q&A / Help] [answered]
I followed the instructions from this page and successful able to switching multiple PHP versions.
Command "php -v" shows correct version of PHP I switched.
However all versions of PHP show on browser with site's folder structure with web page filenames. Navigate to index.php shows PHP code in web page's content.

How to fix this please?

**Comment by shivammathur** [comment, marked as answer]
You need to install a web server. I would recommend installing apache. Follow the Apache Installation section in [this guide.](https://getgrav.org/blog/macos-monterey-apache-multiple-php-versions)

**Reply by chinthyelim** [reply]
Yes, I already have installed the Apache on my Mac Book Pro. It is working when I run command "brew services start httpd". I can see that the DocumentRoot I changed in https.conf loads the site from this specific path. However it shows the PHP code in the localhost's web page's content.

**Reply by chinthyelim** [reply]
Thank you @shivammathur for the [guide](https://getgrav.org/blog/macos-monterey-apache-multiple-php-versions), I missed below section. It is working as charm now!

`<IfModule dir_module>
    DirectoryIndex index.php index.html
</IfModule>

<FilesMatch \.php$>
    SetHandler application/x-httpd-php
</FilesMatch>`
